src/commands/doctor.ts

```diff
@@ -122,9 +127,13 @@ export default class Doctor extends SfdxCommand {
         agent: { https: ProxyAgent(getProxyForUrl(raw)) },
       });
 
-      const title = await this.ux.prompt('Enter a title for your new issue');
+      const title: Record<string, string> = await this.prompt({
```

Comment:
why is the typing necessary?  doesn't `prompt` default to Record<string, string>

Reply:
@mshanemc No, the prompt function returns `<R extends Prompter.Answers>` which is same as `Record<string, unknown>